rviz_ogre_vendor/CMakeLists.txt

```diff
@@ -201,7 +202,7 @@ macro(build_ogre)
     DIRECTORY
       ${CMAKE_CURRENT_BINARY_DIR}/ogre_install/
     DESTINATION
-      ${CMAKE_INSTALL_PREFIX}/opt/rviz_ogre_vendor
+      opt/rviz_ogre_vendor
```

Comment:
This part of the change is actually intended to accomplish the opposite - to support `CMAKE_STAGING_PREFIX` during build of the vendor package itself. That isn't something we're currently trying to do, but it seems appropriate to not needlessly exclude that scenario given that we're relying on it functioning correctly here.